### ISSUE TITLE
[BUG]: Unable to spawn the car at garage

### ISSUE THREAD

**Issue by Hritul** [fkk-cz/noire-public#1219]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

I logged in to game, went to paleto garage, the garage had available vehicle as Infiniti car - I pressed enter - the car did not get spawned. 
I went to the insurance place, got the car insured, went to paleto garage again and tried spawning the car, it wasn't spawned again.
I drove back to insuarance place, restarted the game, got the car insured again, went to paleto again, it showed car name in available vehicle menu but car didn't get spawned and then it was showing there is no vehicles available. I have recorded the last scenario.

So, basically, it happened 2 times on a session and 1 time on another session.

### Expected Behavior

Car should get spawned if it is showing it as available vehicle. But it is not spawning.

### Steps To Reproduce

Car is not spawning, so it can be reproduced on my account. in game name - zero light.

### Anything else?

video link for reference, since my car is not spawning - https://youtu.be/zChcoPz39Ao

**Comment by fkk-cz** [comment]
This vehicle was supposed to be recalled, the `stream/` files were accidentally deleted. Will be fixed on reset.

`[ citizen-server-impl] Asset test_asset/inf.yft uses 32.5 MiB of virtual memory.`
